app/connector/twitter/src/main/resources/META-INF/syndesis/connector/twitter.json

```diff
@@ -77,11 +77,11 @@
               "keywords": {
                 "componentProperty": false,
                 "deprecated": false,
-                "displayName": "Keywords",
+                "displayName": "Query",
                 "group": "common",
                 "javaType": "java.lang.String",
                 "kind": "path",
-                "labelHint": "Multiple search values can be separated with comma.",
+                "labelHint": "Return tweets that match this Twitter-format search expression.",
```

Comment:
How can the user know what is the "Twitter-format" ? WDYT to add an example: camel OR knative ?

Reply:
I did a web search for "Twitter-format search operator" and easily found the info. I don't want to write something that implies that that is the only operator you can specify. But you're right, examples are always helpful. How about: 
Twittter-format search expression, for example: "My Product" returns tweets that contain exactly My Product.

Reply:
That sounds good, thanks Tova.